src/graph_notebook/configuration/generate_config.py

```diff
@@ -64,14 +64,17 @@ def to_dict(self):
 
 class Configuration(object):
     def __init__(self, host: str, port: int,
-                 auth_mode: AuthModeEnum = AuthModeEnum.DEFAULT,
-                 load_from_s3_arn='', ssl: bool = True, aws_region: str = 'us-east-1',
-                 sparql_section: SparqlSection = None, gremlin_section: GremlinSection = None):
+                 auth_mode: AuthModeEnum = DEFAULT_AUTH_MODE,
```

Comment:
Change `DEFAULT_AUTH_MODE` to `DEFAULT_AUTH_MODE.value`, I think this will return the enum object instead of the string value as is.

Reply:
The DEFAULT_AUTH_MODE is of type `AuthModeEnum`, which is what is expected.
Supplying `DEFAULT_AUTH_MODE` would be the same as the original `AuthModeEnum.DEFAULT`. 
It throws an error on my end if I try the .value

Reply:
Ah! My mistake, no problem then.